### PR TITLE
Block programmatic ads by comment page

### PR DESCRIPTION
**Pull request by zeantsoi**
👓 @bsimpson63 @aoiwelle 

Per [reddit-public #5652](https://github.com/reddit/reddit-public/pull/5652).

Accepts an optional `comment` parameter (for comment permalink pages) and blocks programmatic based on its settings.

### CODE REVIEW

**Comment by aoiwelle** [comment]
🐟

File: reddit_adzerk/adzerkads.py [hunk 1]
```diff
@@ -19,14 +19,20 @@
 from reddit_adzerk import adzerkpromote
 
 class Ads(BaseAds):
-    def __init__(self, displayed_things=[], link=None):
+    def __init__(self, displayed_things=[], link=None, comment=None):
```
Comment (bsimpson63) [inline reddit_adzerk/adzerkads.py]: Comment from reddit-public PR:

Could this just take an argument whether or not to allow programmatic ads?

**Comment by bsimpson63** [comment]
💅

**Comment by zeantsoi** [comment]
Closing per [reddit-public #5652](https://github.com/reddit/reddit-public/pull/5652).

**Pull request closed without merging**